ts/components/settings/BlockedList.tsx

```diff
@@ -78,6 +63,7 @@ const BlockedEntries = (props: {
               onSelect={addToSelected}
               onUnselect={removeFromSelected}
               disableBg={true}
+              maxNameWidth={'33vw'}
```

Comment:
I'd rather not have this. Can we not fix it with a flex-shrink on the StyledName element and a maxWidth on the row itself?

Reply:
Part of the issue in this case is that we still want the radio buttons to be at the end of the container. If we limit the width of the entire thing it looks weird 😅

Reply:
<img width="2048" alt="Screenshot 2024-07-26 at 16 07 22" src="https://github.com/user-attachments/assets/67bc9a4e-473b-4e82-a1a6-2ae9c86751b6">

Unless I'm missing something?

Reply:
idk, I'd rather fix this properly :eyes:

Reply:
@Bilb Let's discuss further

Reply:
Discussed and resolved